packages/devtools_app/lib/src/inspector/diagnostics_node.dart

```diff
@@ -65,13 +65,14 @@ class RemoteDiagnosticsNode extends DiagnosticableTree {
 
   final bool isProperty;
 
-  bool get isFlex => getBooleanMember('isFlex', false);
+  bool get isFlex => ['Row', 'Column', 'Flex'].contains(widgetRuntimeType);
```

Comment:
Add a TODO here. It would be nice to have a more robust way of checking for Flex widgets in the future.